### PR TITLE
add an `install` section for DVClive

### PR DESCRIPTION
**Pull request by AlexandreKempf**
related to this discussion: https://iterativeai.slack.com/archives/CB41NAL8H/p1707823158357019

image

### CODE REVIEW

**Review by dberenbaum — CHANGES_REQUESTED** [review]
I think we are missing `[markdown]` here? Otherwise LGTM!

**Review by shcheklein — APPROVED** [review]

**Comment by AlexandreKempf** [comment]
> I think we are missing `[markdown]` here? Otherwise LGTM!

Yeah fair point. To be honest, I'm not sure when you need to do a `[markdown]` install. Is that when you want to generate markdown reports?

**Comment by dberenbaum** [comment]
> > I think we are missing `[markdown]` here? Otherwise LGTM!
> 
> Yeah fair point. To be honest, I'm not sure when you need to do a `[markdown]` install. Is that when you want to generate markdown reports?

Yup, it installs `matplotlib` and is used when `report` is `md` or `notebook`. See [here](https://github.com/iterative/dvclive/blob/aba880710a8cffac1fbb1ed07f974e36961277f3/src/dvclive/live.py#L314-L319).

File: content/docs/dvclive/install.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+$ pip install dvclive
+```
+
+Depending on the type of the [DVClive methods] you plan to use, you might need
+to install optional dependencies:
+
+- install `[image]` to use [`log_image`]
+- install `[plots]` to use [`log_plot`]
+- install `[sklearn]` to use [`log_sklearn_plot`]
+- install `[markdown]` to use the [`make_report`]
```
Comment (dberenbaum) [inline content/docs/dvclive/install.md:19]:
```suggestion
- install `[markdown]` to use [`make_report`] when `report=md` or `report=notebook`
```

**Review by dberenbaum — APPROVED** [review]

**Pull request merged**